src/chttpd/src/chttpd.app.src

```diff
@@ -12,7 +12,7 @@
 
  {application, chttpd, [
     {description, "HTTP interface for CouchDB cluster"},
-    {vsn, git},
+    {vsn, "2.3.0-f18d602"},{id, "f18d602"},
```

Comment:
Isn't committing these to the repo bad practice, since any commit to that tree will require mucking with it and PRs will have commits to this file every time that app is touched?

Reply:
yep this isn't intended, but I'm not sure what has pulled it in.